user/advanced-topics/guivm.md

```diff
@@ -0,0 +1,134 @@
+---
+lang: en
+layout: doc
+permalink: /doc/guivm-configuration/
+ref: 184
+title: GuiVM Configuration
+---
+
+## Gui domain
+
+In this section, we describe how to setup `GuiVM` in several case as described in [GUI Domain](https://www.qubes-os.org/news/2020/03/18/gui-domain/). In all the cases, the base underlying TemplateVM used is `Fedora` with `XFCE` flavor to match current desktop choice in `dom0`. That can be adapted very easily for other desktops and templates. By default, the configured `GuiVM` is a management qube with global admin permissions `rwx` but can be adjusted to `ro` (see [Introducing the Qubes Admin API](https://www.qubes-os.org/news/2017/06/27/qubes-admin-api/)) in pillar data of the corresponding `GuiVM` to setup. Please note that each `GuiVM` has no `NetVM`.
+
+> Note: The setup is done using `SaltStack` formulas with the `qubesctl` tool. When executing it, apply step can take time because it needs to download latest Fedora XFCE TemplateVM and install desktop dependencies.
+
+
+### Hybrid GuiVM `sys-gui`
+
+Here, we describe how to setup `sys-gui` that we call *hybrid mode* or referenced as a *compromised solution* in [GUI Domain](https://www.qubes-os.org/news/2020/03/18/gui-domain/#the-compromise-solution).
+
+[![sys-gui](/attachment/posts/guivm-hybrid.png)](/attachment/posts/guivm-hybrid.png)
+
+In `dom0`, enable the formula for `sys-gui` with pillar data:
+
+```bash
+sudo qubesctl top.enable qvm.sys-gui
+sudo qubesctl top.enable qvm.sys-gui pillar=True
+```
+
+then, execute it:
+
+```bash
+sudo qubesctl --all state.highstate
+```
+
+You can now disable the `sys-gui` formula:
+```bash
+sudo qubesctl top.disable qvm.sys-gui
+```
+
+At this point, you need to shutdown all your running qubes as the `default_guivm` qubes global property has been set to `sys-gui`. In order to use `sys-gui` as GuiVM, you need to logout and select `lightdm` session to *Gui Domain (sys-gui)*. Once logged, you are running `sys-gui` as fullscreen window and you can perform any operation as if you would be in `dom0` desktop.
```

Comment:
I'd add the session type option is in the top right corner, it isn't very obvious otherwise.

Reply:
I will add a screenshot.